projects/validation-messages/src/lib/services/validation-messages.service.ts

```diff
@@ -97,6 +97,43 @@ export class ValidationMessagesService {
     }
   }
 
+  serverErrorsValidator(
+    apiErrors: Observable<ApiErrorMessages> | ApiErrorMessages
```

Comment:
overload method instead of union type
https://www.tutorialsteacher.com/typescript/function-overloading